lib/Cake/Test/Case/Model/Datasource/Database/MysqlTest.php

```diff
@@ -356,41 +356,6 @@ public function testIndexDetection() {
 		$this->assertEquals($expected, $result);
 	}
 
-/**
- * testBuildColumn method
- *
- * @return void
- */
-	public function testBuildColumn() {
-		$restore = $this->Dbo->columns;
-		$this->Dbo->columns = array('varchar(255)' => 1);
```

Comment:
`$this->Dbo->columns` should be array of `'string' => array('name' => 'varchar', 'limit' => '255')` like arrays.

Behaviour with array like `array('varchar(255)' => 1)` is undefined, generates invalid ``'`testName`  DEFAULT NULL COMMENT \'test\''`` (note double space).

Test caused many failures on further tests, as it didn't restored original columns after failure.